src/components/ec-modal/ec-modal.spec.js

```diff
@@ -72,6 +73,30 @@ describe('EcModal', () => {
     expect(wrapper.find('.ec-modal__content').element).toMatchSnapshot();
   });
 
+  it('should have the ec-modal--z-index-X class when the z-index props is given', () => {
```

Comment:
does this statement make sense after last changes?

Reply:
true